src/drivers/distance_sensor/srf02/srf02.cpp

```diff
@@ -37,55 +37,52 @@
  * Driver for the SRF02 sonar range finder adapted from the Maxbotix sonar range finder driver (mb12xx).
  */
 
-#include <px4_config.h>
-#include <px4_defines.h>
-#include <px4_getopt.h>
-#include <px4_workqueue.h>
-#include <containers/Array.hpp>
-
-#include <drivers/device/i2c.h>
-
-#include <sys/types.h>
```

Comment:
Many of these includes probably aren't even needed.

Reply:
I pushed up a commit to delete the includes that are not needed. Thanks!